### PR TITLE
Component body-lock

### PR DESCRIPTION
**Pull request by brock-noah**
### Summary
A component to toggle locking the body, allowing other containers to scroll. Replaces the scrollbar with margin for better experience on Windows.

### Screenshots
body lock

### CODE REVIEW

File: body-lock/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+// or
+<div>
+  <BodyLock lock />
+  <h1 />
+</div>
+```
+
+### `BodyLock` props
+#### `lock : Boolean`
+Wether to lock the body from scrolling.
```
Comment (ben-burlingham) [inline body-lock/README.md]: Whether

**Comment by ben-burlingham** [comment]
What's the use case here?

**Comment by brock-noah** [comment]
When a dropdown is open, the body needs to be locked. A request for the header navigation.

**Comment by brock-noah** [comment]
Changing this up and using just JS. New component soon.

**Pull request closed without merging**